.nvmrc

```diff
@@ -0,0 +1 @@
+12
```

Comment:
Shouldn't we use 14 instead (for docker too)? The issues with node 14 have been fixed in development

Reply:
Have not been running on node 14 myself so far. But I can upgrade & double check everything still works fine then update .nvmrc & the Dockerfile.